### ISSUE TITLE
Provide client information on how to register

### ISSUE THREAD

**Issue by TheOneWithTheBraid** [matrix-org/synapse#10003]
I'm not exactly sure whether it's the right place here to propose, but I didn't find a better one...

**Description:**

Some Matrix servers use external authentication mechanisms, such as LDAP. In this case, Matrix clients are unable to perform a user registration the server does not *seem* to support this. I am looking for a way to work around this:
- the Matrix server should be able to communicate more precise information than that registration is impossible
- I thought of either a *notice* (e.g. a text telling where to register, or that registration is only available for foo bar baz)
- or a direct link to a registration page
- or of course the current information: Registration is impossible

I am unsure whether this is somehow possible at the moment.

**Comment by erikjohnston** [comment]
I don't think it is possible at the moment. This sounds like something that would need to go into the spec, so I'm going to transfer this issue to https://github.com/matrix-org/matrix-doc (from Synapse)
